docs/index.md

```diff
@@ -10,39 +10,47 @@ The HGVS Nomenclature is administered by the [HGVS Variant Nomenclature Committe
 [//]: # (    If you are interested in joining the HVNC and contributing to the maintenance of the HGVS Nomenclature, please see the [Call for Members]&#40;call-for-members.md&#41;.)
 [//]: # (    Applications are due **April 1, 2024**.)
 
-## Contact Us
+<a id="contact-us"></a>
 
-Users of HGVS Nomenclature are invited to contact us to ask questions or get involved with its development.
+## Get Connected with the Community
 
-### Ask questions
+### Sign up for Announcements
+
+Join the [HGVS Nomenclature
+Announcements](https://groups.google.com/g/hgvs-nomenclature-announcements/)
+group and mailing list to receive official administrative announcements from the
+HVNC such as new releases and membership opportunities. This is a very
+low-volume list.
+
+### Engage with the Community
 
 Join the [HGVS Nomenclature](https://groups.google.com/g/hgvs-nomenclature)
-group and mailing list. This is the preferred forum for discussing questions
-and issues regarding the HGVS Nomenclature, and for receiving announcements
-about new releases.
+group and mailing list to discuss questions and issues regarding the HGVS
+Nomenclature.
 
 Messages sent to
 [hgvs-nomenclature@googlegroups.com](mailto:hgvs-nomenclature@googlegroups.com)
 will be distributed to members by email. You do not need to be a member to send
 a message to the group. **However, you will not see a reply unless you join the
-group or manually check the [HGVS Nomenclature](https://groups.google.com/g/hgvs-nomenclature)
-conversation archive**.
+group or manually check the [HGVS
+Nomenclature](https://groups.google.com/g/hgvs-nomenclature) conversation
+archive**.
 
-When members of the community reach out to the HGVS nomenclature committee with
-questions via the google group email, we aim to respond within 2 business days
-of email receipt. If you haven’t received a response within four business days,
-please post a reminder under your question in the google group. For a more
-expedited response in relation to a clinical case, please indicate this in both
-the subject line and body of the email inquiry. We cannot guarantee that even
-clinical inquiries will be addressed outside of business hours or on weekends.
+We aim to respond within two business days of email receipt. If you haven't
```

Comment:
We should not be overwriting this text added in #188 without committee discussion.